setup.py

```diff
@@ -8,6 +12,93 @@ def get_advanced_templates():
+            'gen.build_deploy',
+            'pkgpanda',
+            'pkgpanda.build',
+            'pkgpanda.http',
+            'release',
+            'release.storage']
+    else:
+        return [
+            'dcos_installer',
+            'gen',
+            'gen.build_deploy',
+            'pkgpanda',
+            'pkgpanda.build',
+            'pkgpanda.http',
+            'release',
+            'release.storage',
+            'ssh']
+
+
+def get_package_data():
+    if is_windows:
+        return {
+            'gen': [
+                'ip-detect/aws.sh',
+                'ip-detect/aws_public.sh',
+                'ip-detect/azure.sh',
+                'ip-detect/vagrant.sh',
+                'fault-domain-detect/cloud.sh',
+                'cloud-config-winodws.yaml',
+                'dcos-windows-config.yaml',
+                'dcos-metadata.yaml',
+                'dcos-windows-services.yaml',
+                'aws/dcos-windows-config.yaml',
+                'aws/templates/aws.html',
+                'aws/templates/cloudformation.json',
+                'azure/cloud-config-windows.yaml',
+                'azure/azuredeploy-parameters.json',
+                'azure/templates/acs.json',
+                'azure/templates/azure.html',
+                'azure/templates/azuredeploy.json',
+                'build_deploy/bash/dcos_generate_config.sh.in',
+                'build_deploy/bash/Dockerfile.in',
+                'build_deploy/bash/installer_internal_wrapper.in',
+                'build_deploy/bash/dcos-launch.spec'
+            ] + get_advanced_templates(),
+            'pkgpanda': [
+                'docker.windows/dcos-builder/Dockerfile'
+            ]
+        }
+    else:
+        return {
+            'gen': [
+                'ip-detect/aws.sh',
+                'ip-detect/aws_public.sh',
+                'ip-detect/azure.sh',
+                'ip-detect/vagrant.sh',
+                'fault-domain-detect/cloud.sh',
+                'cloud-config.yaml',
+                'dcos-config.yaml',
+                'dcos-metadata.yaml',
+                'dcos-services.yaml',
+                'aws/dcos-config.yaml',
+                'aws/templates/aws.html',
+                'aws/templates/cloudformation.json',
+                'azure/cloud-config.yaml',
+                'azure/azuredeploy-parameters.json',
+                'azure/templates/acs.json',
+                'azure/templates/azure.html',
+                'azure/templates/azuredeploy.json',
+                'build_deploy/bash/dcos_generate_config.sh.in',
+                'build_deploy/bash/Dockerfile.in',
+                'build_deploy/bash/installer_internal_wrapper.in',
+                'build_deploy/bash/dcos-launch.spec',
+                'coreos-aws/cloud-config.yaml',
+                'coreos/cloud-config.yaml'
+            ] + get_advanced_templates(),
+            'pkgpanda': [
+                'docker/dcos-builder/Dockerfile'
+            ]
+        }
```

Comment:
I don't think we need to be doing different things on Linux and Windows here. We should be packaging up all static files that may be used regardless of platform. Pkgpanda has code in this PR which will use the correct files at runtime.

Reply:
I am a little confused. There are various changes in this list form windows to Linux. There will be more going forwards (all .sh files will be replaced with ps1 files).
why do you think this should not be different?

Reply:
Because the code we're packaging up in this file should be capable of deciding which static files to read at runtime based on the platform. Since we don't need to be packaging up a different set of files per platform, we can do the same thing on all platforms and remove this bit of complexity in the script.

TBH I'm not yet sold on the idea that this `setup.py` should be doing different things on different platforms. I need to research a little bit more. This script is packaging only Python source code (no compiled binaries), and this PR is updating the Python code to be cross-platform, so it seems like this file shouldn't care about platform.

Reply:
I am reverting this file. if nothing breaks for now I will leave it that way.